src/main/java/codeu/controller/UserProfileServlet.java

```diff
@@ -1,41 +1,151 @@
 package codeu.controller;
 
+
+import codeu.model.data.AboutMeMessage;
+
+
+import codeu.model.data.User;
+import codeu.model.store.basic.AboutMeMessageStore;
+import codeu.model.store.basic.UserStore;
 import java.io.IOException;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+
+
 import javax.servlet.ServletException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+import org.kefirsf.bb.BBProcessorFactory;
+import org.kefirsf.bb.TextProcessor;
+
+
 
 /**
  * Servlet implementation class UserProfileServlet
  */
 @WebServlet("/UserProfileServlet")
 public class UserProfileServlet extends HttpServlet {
 	private static final long serialVersionUID = 1L;
-       
-    /**
-     * @see HttpServlet#HttpServlet()
-     */
-    public UserProfileServlet() {
-        super();
-        // TODO Auto-generated constructor stub
-    }
-
-	/**
-	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
-	 */
-	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
The Google Java style guide requires +2 spaces for each indent block:
https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation

Looks like you're using tabs which results in inconsistent formatting.  It's important everyone is using the same formatting so that our editors aren't fighting with each other every time we modify the same files.

This describes how to modify Eclipse to automatically format the way you want:
https://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs

This shows how to select a block of text and auto format:
https://stackoverflow.com/questions/15655126/how-to-auto-format-code-in-eclipse/15655278